cobigen/cobigen-core/src/test/java/com/devonfw/cobigen/unittest/util/ConfigurationUtilTest.java

```diff
@@ -17,33 +17,38 @@
 import com.devonfw.cobigen.impl.util.ConfigurationFinder;
 
 /**
- *
+ * Test suite for Configuration Utility scenarios.
  */
 public class ConfigurationUtilTest {
 
+  /** Temporary files rule to create temporary folders or files */
   @Rule
   public TemporaryFolder tmpFolder = new TemporaryFolder();
 
+  /**
+   * Tests findTemplatesLocation logic Checks if a template jar is located inside the downloaded folder of template-sets
+   * Checks if a template jar can be loaded directly when being set from the .cobigen properties file
+   *
+   * @throws Exception
+   */
   @Test
   public void testFindTemplatesLocation() throws Exception {
 
     restoreSystemProperties(() -> {
       File userHome = this.tmpFolder.newFolder("user-home");
       System.setProperty("user.home", userHome.getAbsolutePath());
       Path defaultCobigenHome = userHome.toPath().resolve(ConfigurationConstants.DEFAULT_HOME_DIR_NAME);
-      Path templatesFolder = defaultCobigenHome.resolve(ConfigurationConstants.TEMPLATES_FOLDER);
+      Path templatesFolder = defaultCobigenHome.resolve(ConfigurationConstants.TEMPLATE_SETS_FOLDER);
       Files.createDirectories(templatesFolder);
       String templatesArtifact = "templates-devon4j-1.0.jar";
 
-      Path templatesProject = templatesFolder.resolve(ConfigurationConstants.COBIGEN_TEMPLATES);
-      Files.createDirectories(templatesProject);
+      Path adaptedTemplatesDirectory = templatesFolder.resolve(ConfigurationConstants.DOWNLOADED_FOLDER);
```

Comment:
You refer to the downloaded folder. Wouldn't `downloadedTemplatesDirectory` be a better naming

Reply:
Adjusted.